Formula/xdelta.rb

```diff
@@ -12,9 +12,25 @@ class Xdelta < Formula
     sha256 "7adf5ae7a00473f5c12f8c377da22ad3f98a0ef4e179c6c0b64b03de075cc756" => :mountain_lion
   end
 
+  depends_on "xz"
```

Comment:
Please shunt this below the `:build` deps. Preferred order is `:build`, mandatory, `:recommended` and then `:optional`. Normally wouldn't bother pointing it out, but since I've other things to highlight might as well mention it.